cmd/crd/cmd/generate.go

```diff
@@ -66,5 +66,8 @@ func GeneratorForFlags(f *flag.FlagSet) *crdgenerator.Generator {
 	// TODO: Do we need this? Is there a possibility that a crd is namespace scoped?
 	f.StringVar(&g.Namespace, "namespace", "", "CRD namespace, treat it as root scoped if not set")
 	f.BoolVar(&g.SkipMapValidation, "skip-map-validation", true, "if set to true, skip generating validation schema for map type in CRD.")
+	f.StringVar(&g.ApisPath, "apis-path", "pkg/apis", "the path to search for apis")
+	f.StringVar(&g.ContextDir, "context-dir", "./pkg/apis", "context directory for api paths")
+	f.StringVar(&g.ApisPkg, "apis-pkg", "", "package to consider as the project api package")
```

Comment:
it's unclear to me why these need to be three different options.  Presumably, `apis-path` and `context-dir` can be a single option, and `apis-pkg` can be automatically inferred from `apis-path`, unless I'm missing something.

Reply:
I think we can combine the `apis-path` and `context-dir`. As per current code, I see `apis-pkg` is optional (inferred by default). Being able to override can be useful in some cases.

Reply:
@droot exactly, I agree that those two can/should be combined (done)